source/scss/components/_ibe-icons.scss

```diff
@@ -0,0 +1,15 @@
+.ibe-type-icons {
+  align-items: center;
+  display: inline-flex;
+  line-height: 26px;
```

Comment:
I should have seen it in the first place. Please use our typography-variables.scss under variables.